src/cryptoadvance/specter/static/hwi.js

```diff
@@ -45,6 +45,22 @@ class HWIBridge {
         return await this.fetch("detect_device", 
             { device_type: type, rescan_devices: rescan });
     }
+
+    async togglePassphrase(device, passphrase="") {
+        /**
+            Tells the server to send the 'togglepassphrase' command to the device.
+            KeepKey and Trezor only.
+        **/
+        if(!('passphrase' in device)){
+            device.passphrase = passphrase;
+        }
```

Comment:
for this command we don't need a passphrase, it's ignored if I understand correctly